README.md

```diff
@@ -1 +1,87 @@
+
+1. **If there are other VC members currently attempting the task challenge**, partner up with them on any help you need or any questions you may have.
+2. **If any VC members have successfully completed the challenge**, review supplemental resources they created after completing the task challenge (if any) and reach out to them with questions.
+3. **If you are the first person to attempt the task challenge**, communicate any questions you have to the VC member who posted the challenge and reach out to them directly for help.
+
+### Submitting Your Technical Task Challenge
+
+Unless you are the first person to attempt the task challenge, you will be required to have another VC member peer review your work before submitting your final solution to the challenge poster. If there are other VC members currently attempting the task challenge, you should pair up and peer review each other's work. If there are no other members currently attempting the task challenge, ask a member who has successfully completed it to review your work.
+
+After your solution has been peer reviewed and necessary changes made, you may proceed to submit it to the challenge originator for final analysis and feedback. Once you receive your feedback from the challenge poster and are told you've successfully completed it, you may create a new pull request to move your name in the technical task challenge `README` from the list of participants to the list of successful completions. Link back to your challenge solution in your fork if you'd like!
+
+#### Example
+
+
+At work, one of my tasks is to implement a CI/CD setup for a React/Node application using GitLab CI and Jenkins. If I wanted to create a task challenge for this, I would fork the repository, add my information to the directory document, and create a new folder called `Implement CI/CD Setup for a React/Node Application Using GitLab CI and Jenkins`. I would add a `README` file with an overview of the challenge and instructions on how to proceed and link that back to the directory document.
+
+Since I've done this before and am very comfortable with the process, I can help whoever tries it out first. Then as more people attempt the challenge, the pool of people who can help answer questions and debug grows, so I don't become overwhelmed with requests for support.
+
+### Going Further
+
+There are a few more ideas for what we can do after a VC member has completed a task challenge. These are completely optional and open to change.
+
+1. The challenge originator can provide them with a list of resources and/or topics to follow up on and expand their newfound knowledge.
+2. The challenge completer can create a resource of their own covering what they just learned. Options include but are not limited to: blog posts, videos/live streams, Twitter spaces, social media posts, podcasts, etc. We would want to link them in the repository, but they can be shared outside the community as well.
```

Comment:
**suggestion (non-blocking):** We should either reword this section, build it out more, or split it up within the rest of the `README.md` file.